### ISSUE TITLE
Black Mage: Blunder on expired element

### ISSUE THREAD

**Issue by VilcsiTH** [Felscream/DragoonMayCry#29]
Having an element expire should never happen to a Black Mage, and when it does, it usually leaves you on 0 mana or ruins the entire rotation, especially now with Flare Star. If it's possible to track, I'd say that would be a good thing to give a blunder to.(I mentioned having an "Easy Mode" in #28, I think this one should still count alongside the vuln up blunder even in Easy mode.)

I think "custom blunders" like these would be fun for jobs, as every job has something the players think of as a blunder, for example, I can think of Red Mage's blunders being losing Mana Stack(from misinputting enhanced actions and not having enough to reach 3, for example) or overloading Mana(crystal turns black or white on the balance gauge). Sadly, I don't play other classes enough to know for sure what their blunders would be, so others would need to chime in on that.

**Comment by Felscream** [comment]
Your red mage suggestion is interesting, but I don't think I'll add it.

I'd prefer to add job specific bonuses and not blunders since depending on the context and fight optimization, some actions that would be considered 'bad' in regular gameplay could actually be beneficial for your damage output in the long run like non-standard lines for BLM in Endwalker or some cartridge shenanigans with GNB.

You can find examples of bonuses already implemented for bard and healers on the 'how it works' window (top right question mark on the plugin window `/dmc`)

For BLM I did not want to track enochian since if you drop it, your spells will be locked and you will naturally drop your uptime, causing a blunder.

**Comment by Felscream** [comment]
They are killing BLM in 7.2 so 🤷‍♂️

**Comment by VilcsiTH** [comment]
Yep, can't wait... This new "Summoner without Summons" class is looking mighty enticing...
Hoping you didn't spend too much time on this issue, and thank you for your work!
